frontend/.angular-cli.json

```diff
@@ -36,12 +36,13 @@
   },
   "lint": [
     {
-      "files": "src/**/*.ts",
-      "project": "src/tsconfig.json"
+      "project": "src/tsconfig.app.json"
```

Comment:
you sure it's not gonna take `node_modules` with it for the linting if you remove the `files` key?